src/Component/Dashboard/DashboardStatistics/DashboardStatistics.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+import {
+  Statistic
+} from 'antd';
+
+type DashboardStatisticsProps = {
+  service: any;
```

Comment:
I would prefer to create the service inside this component. 

Either way it should be typed correctly.